capstone/src/main/java/capstone/Extras/Helper.java

```diff
@@ -236,6 +236,26 @@ public static String customer_search() {
     }
   }
 
+  public static Customer customer_search_new() {
+    while (true) {
+      // for admin and teller use
+      System.out.print("Enter the customer username: ");
+      String username = Helper.readLine();
+      Optional<Customer> queryRes = Database.getCustomer(username);
+      if (!queryRes.isEmpty()) {
```

Comment:
There is a `.isPresent()` function call that is the opposite of `isEmpty()` so you don't need the `!`.